prow/jobs/kyma/kyma-integration.yaml

```diff
@@ -134,9 +135,30 @@ periodics:
         - "bash"
       args:
         - "-c"
-        - "development/disks-cleanup.sh ${CLOUDSDK_CORE_PROJECT}"
+        - "development/disks-cleanup.sh -project=${CLOUDSDK_CORE_PROJECT} -dryRun=true"
+- name: orphaned-clusters-cleaner
+  #cron: "0 */4 * * *" # "Every four hours"
+  cron: "*/10 * * * *" # "Every ten minutes (Temporary Dry Run to ensure it works)"
+  labels:
+    preset-sa-gke-kyma-integration: "true"
+    preset-gc-project-env: "true"
+  decorate: true
+  extra_refs:
+    - org: kyma-project
+      repo: test-infra
+      base_ref: master
+      path_alias: github.com/kyma-project/test-infra
+  spec:
+    containers:
+    - image: eu.gcr.io/kyma-project/prow/buildpack-golang:0.0.1
+      command:
+        - "bash"
+      args:
+        - "-c"
+        - "development/clusters-cleanup.sh -project=${CLOUDSDK_CORE_PROJECT} -dryRun=true"
```

Comment:
`Clusters-cleanup` or `resource-cleanup` should be executed here?

Reply:
clusters-cleanup.sh
resources-cleanup is a script introduced to reduce repetition, it contains common code.